### ISSUE TITLE
Making the plugin available through the plugin manager

### ISSUE THREAD

**Issue by stof** [sebhoerl/Twig-netbeans#10]
Is there a project of making the plugin available through the Netbeans plugin manager ? This would make the update easier and would make the plugin far easier to find.

**Comment by sebhoerl** [comment]
Yes, I can request a review of the plugin on the Netbeans plugin platform. Once it passed the review it will be available in the plugin manager as I understood.
However I want to have embedded highlighting and code completion for CSS and JavaScript before I request the review. I hope to get an answer on the Netbeans mailing list soon or find a solution myself within the next few days.

**Comment by stof** [comment]
What is the status about it ?

**Comment by chesteroni** [comment]
Seems that there is no status :(
And there is no clear instruction so installation with git is complicated :/
